source/includes/options-mongodrdl-commands.yaml

```diff
@@ -0,0 +1,265 @@
+program: mongodrdl
+name: sample
+directive: commandoption
+description: |
+  Samples the namespaces specified by the
+  :option:`--db <mongodrdl --db>`, optional
+  :option:`--collection <mongodrdl --collection>`, and other
+  :ref:`command line options <mongodrdl-command-line-options>` to
+  output a :ref:`.drdl <drdl>` file. The ``sample`` command
+  encapsulates all of :binary:`~bin.mongodrdl` behavior prior to version
+  2.11.
+
+  .. code-block:: sh
+
+     mongodrdl sample --db <db-name> --collection <collection-name> --out <filename>.drdl
+---
+program: mongodrdl
+name: list-schema-ids
+directive: commandoption
+description: |
+  Lists the :ref:`ObjectId <document-bson-type-object-id>` of each
+  stored schema and the date it was created. 
+
+  ``list-schema-ids`` requires the following option:
+  
+  .. list-table::
+     :header-rows: 1
+     :widths: 25 75
+
+     * - Name
+       - Description
+
+     * - ``--schemaSource``
+       - *Required.* Specifies the database where the schema information
+         is stored.
+  
+  For example, the following command lists the schema ids in the
+  ``schemas`` database:
+
+  .. code-block:: sh
+
+     mongodrdl list-schema-ids --schemaSource schemas
+
+  The output resembles the following:
+
+  .. code-block:: sh
+     :copyable: false
+
+     5d72ad695c23a9e3e26e3c85 2019-09-06T15:03:05.556Z
+     5d72adea5c23a9e44882a1ad 2019-09-06T15:05:14.349Z
+
+---
+program: mongodrdl
+name: list-schema-names
+directive: commandoption
+description: |
+  Lists the name and :ref:`ObjectId <document-bson-type-object-id>` for
+  each stored schema that has a name.
+
+  .. include:: /includes/fact-mongodrdl-schema-name.rst
+  
+  ``list-schema-names`` requires the following option:
+  
+  .. list-table::
+     :header-rows: 1
+     :widths: 25 75
+
+     * - Name
+       - Description
+
+     * - ``--schemaSource``
+       - *Required.* Specifies the database where the schema information
+         is stored.
+
+  For example, the following command returns the name and ObjectId for
+  each named schema in the ``schemas`` database:
+
+  .. code-block:: sh
+
+     mongodrdl list-schema-names --schemaSource schemas
+
+  The output resembles the following:
+
+  .. code-block:: sh
+     :copyable: false
+
+     movies 5d72ad695c23a9e3e26e3c85
+     theaters 5d72adea5c23a9e44882a1ad
+---
+program: mongodrdl
+name: upload
+directive: commandoption
+description: |
+  Uploads the schema from the specified ``.drdl`` file to the database
+  specified by the :option:`--schemaSource <mongodrdl --schemaSource>`
+  option. By default, schemas are uploaded to the ``schemas``
+  collection.
```

Comment:
Is it possible to specify a different collection? If not, I'd leave off the "By default" part. Also, is there ever a time when the user needs to specify the collection in which schema information is stored? If not, you could omit this sentence altogether.